draft-packages/table/KaizenDraft/Table/Table.tsx

```diff
@@ -115,6 +115,7 @@ type TableHeaderRowCellProps = OverrideClassName<
   wrapping?: "nowrap" | "wrap"
   align?: "start" | "center" | "end"
   tooltipInfo?: string
+  showTooltipIcon?: boolean
```

Comment:
We prefer our props to be false by default. See https://github.com/cultureamp/kaizen-design-system/blob/main/docs/basic-component.md#props

So I'd go for `isTooltipIconHidden` for this.

Reply:
Thanks @dougmacknz, that's done now.